bundles/org.openhab.core.io.rest.auth/src/main/java/org/openhab/core/io/rest/auth/internal/TokenResource.java

```diff
@@ -154,6 +158,49 @@ public Response getSessions(@Context SecurityContext securityContext) {
         return Response.ok(new Stream2JSONInputStream(sessions)).build();
     }
 
+    @GET
+    @Path("/apitokens")
+    @Operation(summary = "List the API tokens associated to the authenticated user.", responses = {
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = UserApiTokenDTO.class))) })
+    @Produces({ MediaType.APPLICATION_JSON })
+    public Response getApiTokens(@Context SecurityContext securityContext) {
+        if (securityContext.getUserPrincipal() == null) {
+            return JSONResponse.createErrorResponse(Status.UNAUTHORIZED, "User is not authenticated");
```

Comment:
This response seems to be missing from the OpenAPI responses of the `@Operation`.